src/contexts/app.ts

```diff
@@ -516,14 +519,14 @@ class App extends AsyncEventEmitter {
   }
 
   pinAuth() {
-    return new Promise<void>(async (resolve, _reject) => {
+    return new Promise<boolean>(async (resolve, _reject) => {
       const password = await this.getPassword();
 
       const callback = (value: string) => {
         if (password === value) {
           this.off('enterPin', callback);
           this.emit(Events.enterPinSuccess);
-          resolve();
+          resolve(true);
         } else {
           this.emit('errorPin', 'not match');
```

Comment:
Maybe we need to resolve false here?
```suggestion
          this.emit('errorPin', 'not match');
          resolve(false);
```